python_modules/dagster/dagster/_utils/error.py

```diff
@@ -82,7 +83,13 @@ def serializable_error_info_from_exc_info(
     # Whether to forward serialized errors thrown from subprocesses
     hoist_user_code_error: Optional[bool] = True,
 ) -> SerializableErrorInfo:
-    _exc_type, e, _tb = exc_info
+    # `sys.exc_info() return Tuple[None, None, None] when there is no exception being processed. We accept this in
+    # the type signature here since this function is meant to directly receive the return value of
+    # `sys.exc_info`, but the function should never be called when there is no exception to process.
+    exc_type, e, tb = exc_info
+    exc_type = check.not_none(exc_type)
+    e = check.not_none(e)
+    tb = check.not_none(tb)
```

Comment:
Add a helpful string in the `additional_message` parameter of the checks stating that the likely reason that this has been fired is that sys.exc_info() has been called and there is no exception to process.